src/domain/registration/utils.ts

```diff
@@ -678,3 +680,28 @@ export const exportSignupsAsExcel = ({
       addNotification({ type: 'error', label: error.message });
     });
 };
+
+export const getPriceGroupOptionsForPriceGroup = ({
```

Comment:
This could maybe be in a separate refactoring commit together with the changes in `src/domain/registration/formSections/priceGroups/priceGroup/PriceGroup.tsx` and in `src/domain/registration/__tests__/utils.test.ts`